### ISSUE TITLE
Self Sign Cert for VM winrm - Status 400

### ISSUE THREAD

**Issue by rickychew77** [aztfmod/terraform-azurerm-caf#813]
The block of codes below was added under VM's tfvars to enable winrm_listener in VM and create self-signed cert.
```
        winrm = {
          enable_self_signed = true
        }
```
However we get the error below:

`│ Error: keyvault.BaseClient#CreateCertificate: Failure responding to request: StatusCode=400 -- Original Error: autorest/azure: Service returned an error. Status=400 Code="BadParameter" Message="Property  has invalid value\r\n"`

**Comment by arnaudlh** [comment]
hi @jorseng I've assigned you for follow-up!

**Comment by rickychew77** [comment]
#821 issue is on tag limit of 15 in key vault cert

**Comment by arnaudlh** [comment]
thanks @rickychew77 closing this one as #821 has been merged and ramp for 5.5.0 release
